docs/crashlytics/usage.mdx

```diff
@@ -58,6 +58,29 @@ Firebase Crashlytics will upload the crash report to the Firebase Console.
 The error will be show on the Firebase Crashlytics dashboard as an instance of `FirebaseCrashlyticsTestCrash`, with a message of
 `This is a test crash caused by calling .crash() in Dart.`
 
+## Crash types
+
+### Fatal crash
+
+If you would like to record a fatal error, you may pass in a `fatal` argument as `true`.
+The crash report will appear in your Crashlytics dashboard with the event type `Crash`, the event summary stack trace will also be referenced as a `Fatal Exception`.
+
+```dart
+await FirebaseCrashlytics.instance.recordError(error, stackTrace,
```

Comment:
Formatting

Reply:
Not 100% sure what formatting needs to happen? all the other code snippets aren't indented at the beginning of a code block. Nothing changed when I ran prettier either 😅

Reply:
I manually formatted the code.